nixosConfigurations/ponkila-ephemeral-beta/default.nix

```diff
@@ -120,9 +121,11 @@ in {
 
   # Secrets
   sops = {
-    secrets."wireguard/wg0" = {
-      sopsFile = ./secrets/default.yaml;
-    };
+    defaultSopsFile = ./secrets/default.yaml;
+    secrets."wireguard/wg0" = {};
+    secrets."ssvnode/password" = {};
+    secrets."ssvnode/publicKey" = {};
+    secrets."ssvnode/privateKey" = {};
```

Comment:
Nice to see the usage of defaultSopsFile if it is actually working.

Why are these here even though sops isn't actually used?

Reply:
Main idea was to indeed see if it works, and it does. I now updated the config to actually use these values. One upside in general to configure these this way is to ensure that the keys do not get accidentally lost or overridden, given that the keys are found from git now.